test/integration/app.js

```diff
@@ -69,6 +69,74 @@ describe('app', function test() {
+            expect(result.openingTimes.general.monday).to.be.instanceof(Array);
+            expect(result.openingTimes.general.tuesday).to.be.instanceof(Array);
+            expect(result.openingTimes.general.wednesday).to.be.instanceof(Array);
+            expect(result.openingTimes.general.thursday).to.be.instanceof(Array);
+            expect(result.openingTimes.general.friday).to.be.instanceof(Array);
+            expect(result.openingTimes.general.saturday).to.be.instanceof(Array);
+            expect(result.openingTimes.general.sunday).to.be.instanceof(Array);
+            expect(result.outOfHours).to.be.a('string');
+            expect(result.hasEps).to.be.a('boolean');
+            expect(result.address).to.be.instanceof(Object);
+            expect(result.address.line1).to.be.a('string');
+            expect(result.address.line2).to.not.be.undefined;
+            expect(result.address.line3).to.be.a('string');
+            expect(result.address.city).to.be.a('string');
+            expect(result.address.county).to.be.a('string');
+            expect(result.address.postcode).to.be.a('string');
+            expect(result.contacts).to.be.instanceof(Object);
+            expect(result.contacts.website).to.be.a('string');
+            expect(result.contacts.additionalContacts).to.be.instanceof(Array);
+            expect(result.contacts.telephoneNumber).to.be.a('string');
+            expect(result.contacts.isNonGeographicPhoneNumber).to.be.a('boolean');
+            expect(result.contacts.phoneChargeMethod).to.be.a('string');
+            expect(result.contacts.phoneChargeAmount).to.not.be.undefined;
+            expect(result.contacts.telephoneExtension).to.not.be.undefined;
+            expect(result.contacts.fax).to.be.a('string');
+            expect(result.contacts.email).to.not.be.undefined;
+            expect(result.directionsInformation).to.not.be.undefined;
+            expect(result.externalProfileUrl).to.not.be.undefined;
+            expect(result.identifier).to.be.a('string');
+            expect(result.name).to.be.a('string');
+            expect(result.organisationType).to.be.a('string');
+            expect(result.parkingInformation).to.not.be.undefined;
+            expect(result.provider).to.be.a('string');
+            expect(result.summary).to.be.a('string');
+            expect(result.links).to.be.instanceof(Array);
+            expect(result.location).to.be.instanceof(Object);
+            expect(result.location.type).to.be.a('string');
+            expect(result.location.coordinates).to.be.instanceof(Array);
+            expect(result.openingTimesAsOffset).to.be.instanceof(Array);
+            expect(result.openingTimesAlterationsAsOffset).to.be.instanceof(Array);
+            expect(result.distanceInMiles).to.be.a('number');
+            expect(result.openingTimesMessage).to.be.a('string');
+            expect(result.isOpen).to.be.a('boolean');
```

Comment:
should there be a check for 'nextOpen' too, or are some undefined?

Reply:
Yeah, i've added it but not pushed it till now!